litho-processor/src/main/java/com/facebook/litho/specmodels/generator/BuilderGenerator.java

```diff
@@ -733,4 +768,21 @@ private static MethodSpec generateReleaseMethod(SpecModel specModel) {
         .addStatement("$L.release(this)", BUILDER_POOL_FIELD)
         .build();
   }
+
+  private static String generateInitializer(PropDefaultModel propDefaultModel,
```

Comment:
If a line exceeds 100 we put all the parameters on the next line, including the first one 👍